website/docs/cloud-docs/integrations/service-now/service-catalog-terraform/index.mdx

```diff
@@ -92,47 +97,35 @@ It requires the following ServiceNow plugins as dependencies:
-   this Terraform organization. This provider will include one or more
-   repositories with Terraform configurations. Any repository that could be
-   connected to a manually-created HCP Terraform workspace can also be
-   used as a workspace template in the ServiceNow integration.
-1. On your organization's VCS provider settings page (Settings > VCS Providers),
-   find the "OAuth Token ID" for the VCS provider or providers you will be using
-   with the ServiceNow integration. HCP Terraform uses the OAuth token ID to
-   identify and authorize the VCS provider.
-   **Save the OAuth token ID for later.**
-1. Identify one or more VCS repositories in the VCS provider containing
-   Terraform configurations that will be provisioned by the ServiceNow Terraform
-   integration. Take note of any Terraform or environment variables used by the
-   repositories you select.
-   **Save the Terraform and environment variables for later.**
-1. Provide the following information to the ServiceNow Admin.
-1. The organization name
-1. The team API token
-1. The hostname of your Terraform Enterprise instance, or of HCP Terraform.
-   The hostname of HCP Terraform is `app.terraform.io`.
-1. The OAuth token ID(s) of your VCS provider(s), and the repository
-   identifier for each VCS repository containing Terraform configurations that
-   will be used by the integration.
-1. Any Terraform or environment variables required by the configurations in the
+4. If you are using the [VCS flow](/terraform/cloud-docs/integrations/service-now/service-catalog-terraform/service-catalog-config#vcs-configuration): 
+    1. If you haven't yet done so, [connect a VCS
+    provider](/terraform/cloud-docs/vcs) to this Terraform organization. This provider will include one or more repositories with Terraform configurations. Any repository that could be connected to manually-created HCP Terraform workspace can also be used as a workspace template in the ServiceNow integration.
+    2. On your organization's VCS provider settings page (Settings > VCS Providers), find the "OAuth Token ID" for the VCS provider or providers you will be using with the ServiceNow integration. HCP Terraform uses the OAuth token ID to identify and authorize the VCS provider. **Save the OAuth token ID for later.**
+    3. Identify the VCS repositories in the VCS provider containing Terraform configurations that the ServiceNow Terraform integration will deploy. Take note of any Terraform or environment variables used by the repositories you select. Save the Terraform and environment variables for later.
+5. If using the [no-code flow](/terraform/cloud-docs/integrations/service-now/service-catalog-terraform/service-catalog-config#no-code-module-configuration), create one or more no-code modules in the private registry of your HCP Terraform. **Save the no-code module names for later.**
+6. Provide the following information to the ServiceNow Admin:
+    1. The organization name 
```

Comment:
Can all of the following items please be in an unordered list for visual distinction?
```suggestion
    * The organization name 
```